rpc/backend/utils.go

```diff
@@ -380,7 +380,8 @@ func TxExceedBlockGasLimit(res *abci.ResponseDeliverTx) bool {
 	return strings.Contains(res.Log, ExceedBlockGasLimitError)
 }
 
-// TxSuccessOrExceedsBlockGasLimit returns if the tx should be included in json-rpc responses
+// TxSuccessOrExceedsBlockGasLimit returns if the tx should be included in
+// json-rpc responses
```

Comment:
I dont think this comment is accurate. It is true that it is used to determine if it should be included but thats not what the function does. It returns true if the transaction was successful or if it failed with a ExceedBlockGasLimit error. What it is used for is outside of the function's context.